voltdb/src/main/java/org/voltdb/sortedvolttable/package-info.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * 
+ * VoltDBTableSortedMergeWrangler allows you to merge an array of VoltTable
+ * provided by callAllPartitionProcedure.
+ * 
+ * The intended use case is for when you need to issue a multi partition query
+ * but would prefer not to, as you don't need perfect read consistency and would
+ * rather get the individual VoltDB partitions to issue the query independently
+ * and then somehow merge the results.
+ *
+ * @author srmadscience / VoltDB
+ *
+ */
+package org.voltdb.sortedvolttable;
```

Comment:
this should be in `com.yahoo.ycsb.db.voltdb` or a package under that

Reply:
Should be fixed in 0a534e8434d7e854901fbcf825df0403cc985ba3 .